Jetsnack/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-bin.zip
```

Comment:
Is this required? should we move all samples to `6.7.1`?

Reply:
Not required, just updating to the latest. Yes let's add this to the list of bumps to do for all samples when switching to release artifacts.